src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -788,7 +788,8 @@ def define_availability_of_peak_demand_pricing_assets(
 
         availability_in_period = availability_in_period.add(
             pd.Series(1, index=time_period), fill_value=0
-        )
+        ).loc[dict_values[SIMULATION_SETTINGS][TIME_INDEX]]
+        print(f"lenght of availability_in_period: {len(availability_in_period)}")
```

Comment:
```suggestion
```

forgotten print statement?

Reply:
what happens if TIME_INDEX is longer ?

Reply:
The availability timeseries is always generated for a whole year, so TIME_INDEX should never be longer. Of course, when/if we change the timestep to 15 minutes changes will apply, but that is true for everything.